### PR TITLE
Feat/i18n

### PR DESCRIPTION
**Pull request by m0ksem**
closes https://github.com/epicmaxco/vuestic-ui/issues/2434
closes https://github.com/epicmaxco/vuestic-ui/issues/2381

I18n support for Vuestic components props and aria-label attributes.

Need docs after review, but I wrote enough in comments of useI18nConfig composable to make it clear.

Take a look commit by commit.

### CODE REVIEW

File: packages/ui/src/composables/useTranslation.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  return key
+}
+
+export const useTranslation = () => {
+  const { globalConfig, mergeGlobalConfig } = useGlobalConfig()
+
+  const config = computed(() => globalConfig.value.i18n)
+
+  return {
+    /** Translate prop. Translate only if key has `t:` prefix */
```
Comment (m0ksem) [inline packages/ui/src/composables/useTranslation.ts:22]:
```suggestion
    /** Translate prop. Translate only if key has `$t:` prefix */
```

File: packages/ui/src/components/va-time-input/VaTimeInput.vue [hunk 2]
```diff
@@ -40,23 +40,23 @@
...
             v-bind="{ ...iconProps, ...clearIconProps }"
+            :aria-label="t('resetTime')"
             @click.stop="reset"
             @keydown.enter.stop="reset"
             @keydown.space.stop="reset"
           />
           <va-icon
             v-else-if="!$props.leftIcon"
-            aria-label="toggle dropdown"
+            :aria-label="t('toggleDropdown')"
```
Comment (m0ksem) [inline packages/ui/src/components/va-time-input/VaTimeInput.vue:59]: I might miss something to translate. Just run thought :aria-label and props. @aluarius, do you think something else must be translated too?

**Comment by aluarius** [comment]
Closes https://github.com/epicmaxco/vuestic-ui/issues/2381

File: packages/ui/src/components/va-time-input/VaTimeInput.vue (hunk 2 above)
Comment (aluarius) [reply packages/ui/src/components/va-time-input/VaTimeInput.vue:59]: Some aria attributes are set via computed attributes variable. Additional check for 'aria-label' & ariaLabel patterns is needed. Will try to find time next week for this.

All in all, aria attributes and default props values translations is enough probably (can't remember any other cases).

**Review by rustem-nasyrov — APPROVED** [review]

**Review by asvae — APPROVED** [review]
Don't see any issues 👍.

Flat translation strings are a bit troubling, but probably all right.

**Pull request merged**